.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "pypy3.9"]  # "3.11-dev"
+        python-version: ["3.7", "3.8", "3.9", "3.10", "pypy3.9"]  # "3.11-dev"
```

Comment:
Add 3.11?
Add 3.12-dev?

Reply:
True, thanks! :) Since I'm on it already, that's a great idea 👍